woocommerce/woocommerce-admin
# Fix misaligned status column on orders report

Fixes #7623

This PR fixes the misaligned status column by adding the `labelPositionToLeft` prop to the `OrderStatus` component and setting `labelPositionToLeft={ true }` to the `<OrderStatus/>` on the order report table.


### Screenshots
![Screen Shot 2022-01-06 at 13 58 33](https://user-images.githubusercontent.com/4344253/148336502-eff9ca2a-1e11-4551-b0b0-6478301d8194.png)

The orders on the activity panel also use the `<OrderStatus />`. But the label should be on the right side.

![Screen Shot 2022-01-06 at 14 46 05](https://user-images.githubusercontent.com/4344253/148341629-441d402d-fd8a-4627-8587-a4214aa0832a.png)


### Detailed test instructions:

1. Place some orders as a shopper
2. Go to **Analytics > Orders** Screen
3. Observe that the **Status** column is left-aligned properly and the label is on the **left**
4. Go to **Woocommerce > Home**
5. Dismiss the task list to show the activity panel
6. Click to expand the orders panel
6. Observe that the order status is left-aligned properly and the label is on the **right**
